WordPress/Classes/ViewRelated/NUX/NUXButtonViewController.swift

```diff
@@ -52,7 +52,8 @@ class NUXButtonViewController: UIViewController {
     // MARK: - Helpers
 
     private func accessibilityIdentifierForString(_ string: String?) -> String {
```

Comment:
lol I've got another suggestion I missed the first time.

Moving "string" out of the method name such as `accessibilityIdentifier(for string: String?)` would be a more swifty name.